src/GitVersionCore/Configuration/BranchConfigurationCalculator.cs

```diff
@@ -226,7 +227,7 @@ private static BranchConfig ChooseMasterOrDevelopIncrementStrategyIfTheChosenBra
                     };
                 }
             }
-            else if (Regex.IsMatch(chosenBranch.FriendlyName, masterBranchRegex, RegexOptions.IgnoreCase))
+            else if (Regex.IsMatch(chosenBranch.Name.Friendly, masterBranchRegex, RegexOptions.IgnoreCase))
```

Comment:
Can we expose this as `IBranch.IsMain` in a future PR?

```suggestion
            else if (chosenBranch.IsMain)
```